frappe/public/js/frappe/views/kanban/kanban_board.js

```diff
@@ -531,16 +531,28 @@ frappe.provide("frappe.views");
 		function make_dom() {
 			var opts = {
 				name: card.name,
-				title: remove_img_tags(card.title)
+				title: remove_img_tags(card.title),
+				doc: card.doc
 			};
 			self.$card = $(frappe.render_template('kanban_card', opts))
 				.appendTo(wrapper);
 		}
 
 		function render_card_meta() {
 			var html = "";
+
+			// Add estimated due date to the card
+			if (card.doc.exp_end_date) {
```

Comment:
This only works if the DocType has the fieldname `exp_end_date` which is Task in this case. It should be configurable somewhere.

Reply:
Ah, you're right. How would you suggest I handle this? What other DocTypes or views should it cover?

Reply:
@Alchez this is already a part of the configuration

Reply:
https://github.com/frappe/erpnext/blob/develop/erpnext/projects/doctype/task/task_calendar.js#L6

@Alchez can you please fix this so that the design is generic.

Reply:
Hey @rmehta, sorry, I'm a bit confused with the design. What do you need me to do/look at exactly?

Reply:
@Alchez You can create a configuration file per doctype. For e.g: `task_kanban.js` which can store configuration related to Kanban board